cluster-up/cluster/kind-k8s-sriov-1.14.2/config_sriov.sh

```diff
@@ -28,14 +28,12 @@ function deploy_sriov_operator {
   fi
 
   pushd $operator_path
-    # TODO: right now in CI we need to use upstream sriov cni in order to have this
-    # https://github.com/intel/sriov-cni/pull/88 available. This can be removed once the feature will
-    # be merged in openshift sriov operator. We need latest since that feature was not tagged yet
-    sed -i '/SRIOV_CNI_IMAGE/!b;n;c\              value: nfvpe\/sriov-cni' ./deploy/operator.yaml
-    sed -i 's#image: quay.io/openshift/origin-sriov-network-operator$#image: quay.io/openshift/origin-sriov-network-operator:4.2#' ./deploy/operator.yaml
-    sed -i 's#value: quay.io/openshift/origin-sriov-network-config-daemon$#value: quay.io/openshift/origin-sriov-network-config-daemon:4.2#' ./deploy/operator.yaml
+    export RELEASE_VERSION=4.4
```

Comment:
Indentation looks odd here. Or is it recommended to indent within `pushd` and `popd`?

Reply:
It is kept from the previous version of the code. I personally prefer to indent to make it clear what is the context of pushd.